gprofiler/metadata/application_identifiers.py

```diff
@@ -273,6 +278,17 @@ def get_app_id(self, process: Process) -> Optional[str]:
         return None
 
 
+class _NodeModuleApplicationIdentifier(_ApplicationIdentifier):
+    def get_app_id(self, process: Process) -> Optional[str]:
+        if not _is_node_bin(_get_cli_arg_by_index(process.cmdline(), 0)):
```

Comment:
If you call it only if node app, you don't need to do this check here. In any case we have `is_node_process` for it so you don't need the new code here